src/app/containers/MostRead/utilities/processMostRead.js

```diff
@@ -14,14 +51,30 @@ const mostReadItems = ({ data, numberOfItems }) => {
   // Do not show most read if lastRecordUpdated is greater than 35min as this means PopAPI has failed twice
   // in succession. This suggests ATI may be having issues, hence risk of stale data.
   if (isTest || mostReadRecordIsFresh(data.lastRecordTimeStamp)) {
-    return records
-      .slice(0, numberOfItems)
-      .map(({ id, promo: { headlines, locators, timestamp } }) => ({
-        id,
-        title: headlines.shortHeadline,
-        href: locators.assetUri,
-        timestamp,
-      }));
+    const items = [];
+    for (let i = 0; i < records.length; i += 1) {
+      const mostReadItemData = getMostReadItemData(records[i]);
+      const { href, title } = mostReadItemData;
+      if (href && title) {
+        items.push(mostReadItemData);
+      } else {
+        logger.info(
+          JSON.stringify(
```

Comment:
Could you log this using the same format as we use here?: https://github.com/bbc/simorgh/blob/67846f4ba9cbec71b5a4f6c79dbb08be84483f31/src/app/containers/RadioSchedule/utilities/processRadioSchedule.js#L28?

The logger has been used incorrectly elsewhere in the codebase so it's a little confusing!